### PR TITLE
Feature: summary template

### PR DESCRIPTION
**Pull request by vadymmarkov**
@hyperoslo/everyone 
Here we introduce a template for writing technical reviews or summaries. The idea is to copy a template and fill in the form. Using a template would make it easier to keep all the summaries similar to each other.

### CODE REVIEW

**Comment by onmyway133** [comment]
@vadymmarkov Should we use PR templating for this https://github.com/blog/2111-issue-and-pull-request-templates ?

**Comment by vadymmarkov** [comment]
@onmyway133 PR template is for PR's description. And this template is for MD files.

File: README.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ inspire and push us forward.
 
 We would love you be involved in **Hyper Radar** evolution!
 Come up with new ideas, play with new tech, participate in discussions and
-don't be afraid to express your own views and opinions. Have your built
+don't be afraid to express your own views and opinions. Have you built
```
Comment (sindrenm) [inline README.md:101]: :tada:

File: Lab/template.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
+<!-- Thanks for contributing to Hyper Radar! 🔭 -->
+
+<!--
+Have you built something cool? A library, side project or a homemade
+spacecraft? Then you are in the right place!
```
Comment (sindrenm) [inline Lab/template.md:5]: :rocket:

File: Lab/template.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+Have you tried something promising? Share your experience to help in our
```
Comment (sindrenm) [inline Lab/template.md]: Should we ditch the “in” at the end here?

File: Lab/template.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+community to stay sober and make the right decision when it comes to
+choosing a new technology.
+
+There are a few things to keep in mind when you prepare your text:
+- Choose the right category or create a new folder (if it's something
+completely new).
```
Comment (sindrenm) [inline Lab/template.md]: Mind if we indent this line to line up under “Choose” above?

File: Lab/template.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+-->
+
+# Title
+
+* Website: https://github.com/hyperoslo
+* Documentation: https://github.com/hyperoslo
+* Demo: https://github.com/hyperoslo
+* Categories: NLP, Bot engines, ...
+* Platform: Web, mobile, ...
+* License:	MIT
```
Comment (sindrenm) [inline Lab/template.md]: Shouldn't this be a space?

**Review by sindrenm — CHANGES_REQUESTED** [review]
Nice template! But should we have a one-size-fits-all template, and if so, does this one fit all?

File: Lab/template.md (hunk 2 above)
Comment (onmyway133) [reply Lab/template.md:5]: I love to see @timkurvers 's warcraft game here

Comment (timkurvers) [reply Lab/template.md:5]: Can do! (later)

**Comment by vadymmarkov** [comment]
@sindrenm I think this template is more for reviewing open source tools, frameworks, different kind of services, like for example https://flutter.io (It's a task for you btw 😛). One can potentially write an article on Medium and then make a short summary out of it to publish here under an appropriate category in `Lab`. On the other hand,  it might make more sense to leave `Lab` only for tools and create more folders, for example `Playground` for things like @timkurvers 's warcraft game, with it's own template and set of questions. What do you think @sindrenm?

**Comment by sindrenm** [comment]
Yeah, that sounds reasonable, @vadymmarkov! And thanks for fixing up the indentation. :tada:

**Review by sindrenm — APPROVED** [review]

**Pull request merged**